src/initial-submission/components/DetailsForm.tsx

```diff
@@ -21,6 +21,19 @@ interface Props {
     ButtonComponent?: (props: { saveFunction?: Function }) => JSX.Element;
 }
 
+const useSuggestions = (values: ManuscriptDetails, suggestions: Array<Suggestion>): ManuscriptDetails => {
```

Comment:
Is this a hook? if so it should be in the hook folder, if not i would rename it to avoid confusion